mlrun/utils/helpers.py

```diff
@@ -1259,6 +1259,10 @@ def _fill_project_path_template(artifact_path, project):
     return artifact_path
 
 
+def select_non_empty_fields(input_dict: dict) -> dict:
```

Comment:
```suggestion
def to_non_empty_values_dict(input_dict: dict) -> dict:
```
perhaps says more about what it does?